File: Day 15/volumeCalc.c
#include <stdio.h>
#include <math.h>

void sphereVol(){

    float r;
    printf("Type the sphere radius: \n");
    scanf("%f", &r);

    float vol = 4/3 * M_PI * pow(r, 3);

    printf("The sphere volume is %f \n", vol);

}
void cubeVol(){

    float a;
    printf("Type the cube edge: \n");
    scanf("%f", &a);

    float vol = pow(a, 3);

    printf("The cube volume is %f \n", vol);

}
void cylinderVol(){

    float r;
    float h;
    printf("Type the base radius: \n");
    scanf("%f", &r);
    printf("Type the cylinder height: \n");
    scanf("%f", &h);

    float area = M_PI * pow(r,2);

    float vol = area * h;

    printf("The cylinder volume is %f \n", vol);

}

int main()
{
    sphereVol();
    cubeVol();
    cylinderVol();

    return 0;
}

File: Day 15/volumeCalc.js
function sphereVol(){

    let r = prompt("Type the sphere radius:")

    let vol = 4/3*Math.PI*Math.pow(r,3)

    console.log(`The volume is ${vol}`)

}

function cubeVol(){

    let a = prompt("Type the cube edge:")

    let vol = Math.pow(a,3)

    console.log(`The volume is ${vol}`)

}

function cylinderVol(){

    let r = prompt("Type the base radius:")
    let h = prompt("Type the cylinder height:")

    let area = Math.PI*Math.pow(r,2)

    let vol = area * h

    console.log(`The volume is ${vol}`)

}

cylinderVol()
sphereVol()
cubeVol()
